wazo_calld/plugins/calls/exceptions.py

```diff
@@ -32,6 +32,16 @@ def __init__(self, call_id):
         )
 
 
+class CallOriginUnavailableError(APIException):
+    def __init__(self, call_id):
+        super().__init__(
+            status_code=400,
+            message="Could not connect call: Could not dial caller's requested line",
+            error_id='call-origin-unavailable-error',
```

Comment:
```suggestion
            error_id='call-origin-unavailable',
```
no need to include '-error' suffix in 'error_id' keys